spaceship/views.py

```diff
@@ -288,14 +307,31 @@ def confirm_token(token, expiration=3600):
 
 @app.route('/register', methods=['GET', 'POST'])
 def register():
+  oauth_user_info = session.pop('oauth_user_info', None)
+  session['oauth_next_url'] = url_for('register')
```

Comment:
similar to login, should we only set this when we render the template?

Reply:
done